tests/framework/validators/ExistValidatorTest.php

```
@@ -295,6 +295,68 @@ public function testSecondTargetAttributeWithError(): void
         $customer->clearErrors();
         $customer->addError('name', 'error');
         $validator->validateAttribute($customer, 'email');
-        $this->assertTrue($customer->hasErrors('email')); // validator should not be skipped
+        $this->assertTrue($customer->hasErrors('email'));
+    }
+
+    public function testTargetRelationWithFilterArray(): void
+    {
+        $val = new ExistValidator(['targetRelation' => 'references', 'filter' => ['a_field' => 'ref_to_2']]);
+        $m = ValidatorTestMainModel::findOne(2);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\yiiunit\data\validators\models\ValidatorTestMainModel' in method 'testTargetRelationWithFilterArray'. (undefined)

(StaticAccess)